cdk/domino_cdk/config/iam.py

```diff
@@ -338,6 +338,7 @@ def do_cf():
             "ec2:DescribeAvailabilityZones",
             "ec2:DescribeImages",
             "ec2:DescribeInstances",
+            "ec2:DescribeInstanceStatus",
```

Comment:
This is actually the deployment policy, not the policy running on the nodes, so I don't think it's actually needed here?

Reply:
Ah ok, Reverting here.

Reply:
This was actually the opposite reversion that happened (the one this comment is one should go away, the other one should come back. Sorry. :(

Reply:
My bad, sorry, I read it correctly earlier, not sure what I was thinking.